changepass.pl

```diff
@@ -3,6 +3,8 @@
 # Script for the user to change their webmin password
 
 # Check command line arguments
+do "./miniserv.pl";
```

Comment:
I don't think this is a good way of bringing in the needed functions from miniserv.pl, because it will execute al the main code! Best to split them into a separate file.

Reply:
Okay, understood! Thanks. Here is the final fix. https://github.com/webmin/webmin/commit/d1d475bbd14a294f04ebf4636b7057814eded9a7